tests/intg/test_slide_gen_chain.py

```
@@ -0,0 +1,258 @@
+            "title_slide": {"name": "title_slide", "params": ["title", "subtitle"]},
+            "content_slide": {"name": "content_slide", "params": ["content"]},
+        }
+
+        # Mock slide generation functions
+        def mock_title_slide(**kwargs):
+            return f"# {kwargs.get('title', 'Default Title')}\n## {kwargs.get('subtitle', 'Default Subtitle')}"
+
+        def mock_content_slide(**kwargs):
+            return f"## Content\n{kwargs.get('content', 'Default content')}"
+
+        mock_get_function.side_effect = lambda template_id, func_name: {
```

Ruff (0.12.2)

Unused lambda argument: `template_id`

(ARG005)